detections/cloud/abnormally_high_number_of_cloud_security_group_api_calls.yml

```diff
@@ -53,7 +53,20 @@ tags:
   - All_Changes.object_category
   - All_Changes.status
   - All_Changes.user
-  risk_object: user
-  risk_object_type: user
-  risk_score: 25
   security_domain: network
+  impact: 30
+  confidence: 50
+  # (impact * confidence)/100
+  risk_score: 15
+  context:
+  - Source:Cloud Data
+  - Scope:Inbound
+  - Outcome:Allowed
```

Comment:
@patel-bhavin this all  looks good.

A comment/question for all updated detections:
- why would we re-tag things as Outcome:Allowed?

Reply:
I tagged it as allowed since these searches detected successful attempts